### PR TITLE
fix: Update @expo/config-plugins to ^5.x

### PR DESCRIPTION
**Pull request by c-hristian**
Update @expo/config-plugins to ^5.x to git rid of expo warnings for consuming apps

Test Plan: successful yarn install for both main repo and example repo. ran android example to ensure app was available and login functionality still persisted.

image

### CODE REVIEW

**Comment by jpalumickas** [comment]
I also tested this in local, everything working fine.

**Review by mikehardy — APPROVED** [review]
This looks good to me, based entirely on the testing, really really appreciated.
I've been vocal on this repo and others that I cannot support Expo and we rely on community contributions, I will be equally vocal in saying that I will truly appreciate those contributions and will merge them and release them as quickly as possible if they come in.

Cheers

**Review by mikehardy — CHANGES_REQUESTED** [review]
This will likely be something subtle and only tangentially related, but CI needs to go green before I can merge it :weary:

**Comment by oliuradu** [comment]
Any updates?

**Comment by mikehardy** [comment]
@oliuradu it's open source: you see no updates, there are no updates. Feel free to take the PR, patch it up, and push it as a new PR if you need it. I'm happy to merge things when they appear to be working

**Pull request closed without merging**
